packages/vulkan_tools.rb

```diff
@@ -6,21 +6,21 @@
 class Vulkan_tools < Package
```

Comment:
Use CMake buildsystem?

Reply:
This doesn't really work. It's too complicated, as it is a dual build...